tutorials/README.md

```diff
@@ -1,5 +1,5 @@
 # Cortex Certifai Tutorials 
-This directory contains self contained tutorials which walk through basic parts of Certifai funcitonality 
+This directory contains self contained tutorials which walk through basic parts of Certifai functionality 
```

Comment:
Change to: "This directory contains tutorials that walk through basic Certifai functionality."

Reply:
Done.